### PR TITLE
Trusted IP Ranges (spaces:inboundruleset rename)

### PR DESCRIPTION
**Pull request by amerine**
We are going to launch the inbound access restrictions with the name
"Trusted IP Ranges", as well as drop the "Default Action" logic from the
system. This PR does that.

cc/ @daneharrigan @jesperfj @ryanbrainard

### CODE REVIEW

File: commands/trusted-ips/add.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function* run (context, heroku) {
+  let lib = require('../../lib/trusted-ips')(heroku);
+  let space = context.flags.space;
+  let ruleset = yield lib.getRules(space);
+  ruleset.rules = ruleset.rules || [];
+  if (ruleset.rules.find(rs => rs.source === context.args.source)) throw new Error(`A rule already exists for ${context.args.source}.`);
+  if (ruleset.rules.length === 0) yield cli.confirmApp(space, context.flags.confirm, `Traffic from ${cli.color.red(context.args.source)} will be allowed to make web requests to web dynos, all other inbound traffic denied.`);
+  ruleset.rules.push({action: 'allow', source: context.args.source});
+  ruleset = yield lib.putRules(space, ruleset);
+  cli.log(`Added ${context.args.source} to trusted IP ranges on ${space}`);
```
Comment (dickeyxxx) [inline commands/trusted-ips/add.js]: As a suggestion, you could instead show a bit of progress with `cli.action()`:

```js
let request = lib.putRules(space, ruleset);
yield cli.action(`Adding ${context.args.source} to trusted IP ranges on ${space}`, request);
```

**Comment by dickeyxxx** [comment]
lgtm. If it were me I might add in a bit of cyan to highlight keywords like the source or maybe the space you're working on.

**Comment by amerine** [comment]
@dickeyxxx Do you have a good example I can follow for coloring the keywords?

**Comment by dickeyxxx** [comment]
https://github.com/heroku/heroku-apps/blob/master/commands/stack/set.js#L18

File: commands/trusted-ips/add.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+
+function* run (context, heroku) {
+  let lib = require('../../lib/trusted-ips')(heroku);
+  let space = context.flags.space;
+  let ruleset = yield lib.getRules(space);
+  ruleset.rules = ruleset.rules || [];
+  if (ruleset.rules.find(rs => rs.source === context.args.source)) throw new Error(`A rule already exists for ${context.args.source}.`);
```
Comment (ryanbrainard) [inline commands/trusted-ips/add.js]: do we do anything smart about overlapping ranges? i'm guessing that's handled on the API side?

File: commands/trusted-ips/add.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+let cli = require('heroku-cli-util');
+let co  = require('co');
+
+function* run (context, heroku) {
```
Comment (ryanbrainard) [inline commands/trusted-ips/add.js]: nitpick: can we break some of these onto multiple lines? the conditionals are a bit hard to read and the lines are kind of long.

**Comment by ryanbrainard** [comment]
:+1: A touch of cyan and this will be golden :art:

File: commands/trusted-ips/add.js (hunk 2 above)
Comment (amerine) [inline commands/trusted-ips/add.js]: @ryanbrainard It its checked on the API side.

File: commands/trusted-ips/add.js (hunk 3 above)
Comment (amerine) [inline commands/trusted-ips/add.js]: I agree, I was just following the style that was there before me.

**Comment by amerine** [comment]
color!

Color!

Thanks for the help @dickeyxxx @ryanbrainard

**Comment by dickeyxxx** [comment]
looks great!

**Pull request merged**